multi-agent/config/app_config.py

```
@@ -14,30 +14,17 @@ class AppConfig(SharedConfig):
     version: str = "1.0.0"
     admin_allowed_users: list = []  # Populate with user_ids (usernames) to grant admin access
     secret_key: str = ""
+    # Storage
+    shared_storage: str = "/tmp"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)